src/controllers/AbstractRestController.ts

```diff
@@ -0,0 +1,29 @@
+import { inject } from 'inversify';
+import { BaseHttpController, httpGet, controller } from 'inversify-express-utils';
+import { RedisService } from '../services/RedisService';
+import { TYPES } from '../InjectableTypes';
+
+@controller('/')
+export abstract class AbstractRestController extends BaseHttpController {
```

Comment:
The name 'AbstractRestController' doesn't really describe it's content

Reply:
Maybe `HealthCheckController`?

Reply:
Sounds good 👍